[Usage]: npu-910b 安装vllm和vllm-ascend成功后，可以直接多卡推理模型，但是不能使用Online Serving on Multi Machine

npu-smi 24.1.rc2.1               Version: 24.1.rc2.1 
cann版本是8.0.0
python 3.10
torch2.51
使用https://vllm-ascend.readthedocs.io/en/latest/tutorials.html 中Online Serving on Multi Machine进行单机部署服务时报错

>>> remove /work/share/zyj/inference/logs
INFO 02-26 09:45:07 __init__.py:28] Available plugins for group vllm.platform_plugins:
INFO 02-26 09:45:07 __init__.py:30] name=ascend, value=vllm_ascend:register
INFO 02-26 09:45:07 __init__.py:32] all available plugins for group vllm.platform_plugins will be loaded.
INFO 02-26 09:45:07 __init__.py:34] set environment variable VLLM_PLUGINS to control which plugins to load.
INFO 02-26 09:45:07 __init__.py:42] plugin ascend loaded.
INFO 02-26 09:45:08 __init__.py:28] Available plugins for group vllm.platform_plugins:
INFO 02-26 09:45:08 __init__.py:30] name=ascend, value=vllm_ascend:register
INFO 02-26 09:45:08 __init__.py:32] all available plugins for group vllm.platform_plugins will be loaded.
INFO 02-26 09:45:08 __init__.py:34] set environment variable VLLM_PLUGINS to control which plugins to load.
INFO 02-26 09:45:08 __init__.py:42] plugin ascend loaded.
INFO 02-26 09:45:08 __init__.py:187] No platform detected, vLLM is running on UnspecifiedPlatform
WARNING 02-26 09:45:08 _custom_ops.py:19] Failed to import from vllm._C with ModuleNotFoundError("No module named 'vllm._C'")
INFO 02-26 09:45:08 __init__.py:174] Platform plugin ascend is activated
INFO 02-26 09:45:09 api_server.py:838] vLLM API server version 0.7.1
INFO 02-26 09:45:09 api_server.py:839] args: Namespace(host=None, port=8891, uvicorn_log_level='info', allow_credentials=False, allowed_origins=['*'], allowed_methods=['*'], allowed_headers=['*'], api_key=None, lora_modules=None, prompt_adapters=None, chat_template=None, chat_template_content_format='auto', response_role='assistant', ssl_keyfile=None, ssl_certfile=None, ssl_ca_certs=None, ssl_cert_reqs=0, root_path=None, middleware=[], return_tokens_as_token_ids=False, disable_frontend_multiprocessing=True, enable_request_id_headers=False, enable_auto_tool_choice=False, enable_reasoning=False, reasoning_parser=None, tool_call_parser=None, tool_parser_plugin='', model='/work/share/weights/Qwen2.5-Coder-32B-Instruct/', task='auto', tokenizer=None, skip_tokenizer_init=False, revision=None, code_revision=None, tokenizer_revision=None, tokenizer_mode='auto', trust_remote_code=True, allowed_local_media_path=None, download_dir=None, load_format='auto', config_format=<ConfigFormat.AUTO: 'auto'>, dtype='auto', kv_cache_dtype='auto', max_model_len=26240, guided_decoding_backend='xgrammar', logits_processor_pattern=None, distributed_executor_backend='ray', pipeline_parallel_size=1, tensor_parallel_size=8, max_parallel_loading_workers=None, ray_workers_use_nsight=False, block_size=None, enable_prefix_caching=None, disable_sliding_window=False, use_v2_block_manager=True, num_lookahead_slots=0, seed=0, swap_space=4, cpu_offload_gb=0, gpu_memory_utilization=0.9, num_gpu_blocks_override=None, max_num_batched_tokens=None, max_num_seqs=None, max_logprobs=20, disable_log_stats=True, quantization=None, rope_scaling=None, rope_theta=None, hf_overrides=None, enforce_eager=True, max_seq_len_to_capture=8192, disable_custom_all_reduce=False, tokenizer_pool_size=0, tokenizer_pool_type='ray', tokenizer_pool_extra_config=None, limit_mm_per_prompt=None, mm_processor_kwargs=None, disable_mm_preprocessor_cache=False, enable_lora=False, enable_lora_bias=False, max_loras=1, max_lora_rank=16, lora_extra_vocab_size=256, lora_dtype='auto', long_lora_scaling_factors=None, max_cpu_loras=None, fully_sharded_loras=False, enable_prompt_adapter=False, max_prompt_adapters=1, max_prompt_adapter_token=0, device='auto', num_scheduler_steps=1, multi_step_stream_outputs=True, scheduler_delay_factor=0.0, enable_chunked_prefill=None, speculative_model=None, speculative_model_quantization=None, num_speculative_tokens=None, speculative_disable_mqa_scorer=False, speculative_draft_tensor_parallel_size=None, speculative_max_model_len=None, speculative_disable_by_batch_size=None, ngram_prompt_lookup_max=None, ngram_prompt_lookup_min=None, spec_decoding_acceptance_method='rejection_sampler', typical_acceptance_sampler_posterior_threshold=None, typical_acceptance_sampler_posterior_alpha=None, disable_logprobs_during_spec_decoding=None, model_loader_extra_config=None, ignore_patterns=[], preemption_mode=None, served_model_name=None, qlora_adapter_name_or_path=None, otlp_traces_endpoint=None, collect_detailed_traces=None, disable_async_output_proc=False, scheduling_policy='fcfs', override_neuron_config=None, override_pooler_config=None, compilation_config=None, kv_transfer_config=None, worker_cls='auto', generation_config=None, override_generation_config=None, enable_sleep_mode=False, calculate_kv_scales=False, disable_log_requests=True, max_log_len=None, disable_fastapi_docs=False, enable_prompt_tokens_details=False)
INFO 02-26 09:45:19 config.py:526] This model supports multiple tasks: {'generate', 'reward', 'embed', 'classify', 'score'}. Defaulting to 'generate'.
INFO 02-26 09:45:19 importing.py:14] Triton not installed or not compatible; certain GPU-related functions will not be available.
INFO 02-26 09:45:19 llm_engine.py:232] Initializing a V0 LLM engine (v0.7.1) with config: model='/work/share/weights/Qwen2.5-Coder-32B-Instruct/', speculative_config=None, tokenizer='/work/share/weights/Qwen2.5-Coder-32B-Instruct/', skip_tokenizer_init=False, tokenizer_mode=auto, revision=None, override_neuron_config=None, tokenizer_revision=None, trust_remote_code=True, dtype=torch.bfloat16, max_seq_len=26240, download_dir=None, load_format=auto, tensor_parallel_size=8, pipeline_parallel_size=1, disable_custom_all_reduce=False, quantization=None, enforce_eager=True, kv_cache_dtype=auto,  device_config=npu, decoding_config=DecodingConfig(guided_decoding_backend='xgrammar'), observability_config=ObservabilityConfig(otlp_traces_endpoint=None, collect_model_forward_time=False, collect_model_execute_time=False), seed=0, served_model_name=/work/share/weights/Qwen2.5-Coder-32B-Instruct/, num_scheduler_steps=1, multi_step_stream_outputs=True, enable_prefix_caching=False, chunked_prefill_enabled=False, use_async_output_proc=True, disable_mm_preprocessor_cache=False, mm_processor_kwargs=None, pooler_config=None, compilation_config={"splitting_ops":[],"compile_sizes":[],"cudagraph_capture_sizes":[],"max_capture_size":0}, use_cached_outputs=False, 
2025-02-26 09:45:20,254 INFO HYPERLINK https://worker.py:1654 normalLink \tdfe -10 \tdlt text \tdlf FromInput \tdsub normalLink \tdkey dbqkpfwworker.py:1841 -- Connecting to existing Ray cluster at address: 10.240.34.21:6379...
2025-02-26 09:45:20,265 INFO worker.py:1841 -- Connected to Ray cluster.
INFO 02-26 09:45:24 ray_distributed_executor.py:153] use_ray_spmd_worker: False
(pid=118852) INFO 02-26 09:45:30 __init__.py:28] Available plugins for group vllm.platform_plugins:
(pid=118852) INFO 02-26 09:45:30 __init__.py:30] name=ascend, value=vllm_ascend:register
(pid=118852) INFO 02-26 09:45:30 __init__.py:32] all available plugins for group vllm.platform_plugins will be loaded.
(pid=118852) INFO 02-26 09:45:30 __init__.py:34] set environment variable VLLM_PLUGINS to control which plugins to load.
(pid=118852) INFO 02-26 09:45:30 __init__.py:42] plugin ascend loaded.
(pid=118852) INFO 02-26 09:45:31 __init__.py:187] No platform detected, vLLM is running on UnspecifiedPlatform
(pid=118852) WARNING 02-26 09:45:31 _custom_ops.py:19] Failed to import from vllm._C with ModuleNotFoundError("No module named 'vllm._C'")
(pid=118852) INFO 02-26 09:45:31 __init__.py:174] Platform plugin ascend is activated
(RayWorkerWrapper pid=118835) WARNING 02-26 09:45:32 utils.py:547] Overwriting environment variable ASCEND_RT_VISIBLE_DEVICES from '1' to '0,1,2,3,4,5,6,7'
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572] Error executing method 'init_device'. This might cause deadlock in distributed execution.
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572] Traceback (most recent call last):
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm/worker/worker_base.py", line 564, in execute_method
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]     return run_method(target, method, args, kwargs)
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm/utils.py", line 2208, in run_method
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]     return func(*args, **kwargs)
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm_ascend/worker.py", line 173, in init_device
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]     current_platform.set_device(self.device)
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm_ascend/platform.py", line 83, in set_device
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]     torch.npu.set_device(device)
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/torch_npu/npu/utils.py", line 58, in set_device
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]     torch_npu._C._npu_setDevice(device_id)
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572] RuntimeError: Initialize:build/CMakeFiles/torch_npu.dir/compiler_depend.ts:226 NPU function error: c10_npu::SetDevice(device_id_), error code is 107001
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572] [ERROR] 2025-02-26-09:45:37 (PID:118835, Device:0, RankID:-1) ERR00100 PTA call acl api failed
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572] [Error]: Invalid device ID.
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         Check whether the device ID is valid.
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572] EE1001: [PID: 118835] 2025-02-26-09:45:34.406.377 The argument is invalid.Reason: Set visible device failed, invalid device=1, input visible devices:1
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         Solution: 1.Check the input parameter range of the function. 2.Check the function invocation relationship.
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         TraceBack (most recent call last):
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         Failed to drvGetPlatformInfo, ret=4[FUNC:DrvGetPlatformInfo][FILE:ai_drv_dev_api.cpp][LINE:79]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         halGetDeviceInfo get device type version failed , ret:4[FUNC:Init][FILE:config_manager.cpp][LINE:61]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         The argument is invalid.Reason: Set visible device failed, invalid device=1, input visible devices:1
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         rtSetDevice execute failed, reason=[device id error][FUNC:FuncErrorReason][FILE:error_message_manage.cc][LINE:53]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         open device 1 failed, runtime result = 107001.[FUNC:ReportCallError][FILE:log_inner.cpp][LINE:161]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         The argument is invalid.
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         Failed to open device, retCode=0x7020014, deviceId=1.[FUNC:InitRawDriver][FILE:raw_device.cc][LINE:123]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         Failed to init RawDriver, device_id=1, retCode=0x7020014[FUNC:Init][FILE:raw_device.cc][LINE:214]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         Check param failed, dev can not be NULL![FUNC:DeviceRetain][FILE:runtime.cc][LINE:4052]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         Check param failed, dev can not be NULL![FUNC:PrimaryContextRetain][FILE:runtime.cc][LINE:3749]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         Check param failed, ctx can not be NULL![FUNC:PrimaryContextRetain][FILE:runtime.cc][LINE:3776]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         Check param failed, context can not be null.[FUNC:NewDevice][FILE:api_impl.cc][LINE:2456]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         New device failed, retCode=0x7010006[FUNC:SetDevice][FILE:api_impl.cc][LINE:2480]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         rtSetDevice execute failed, reason=[device retain error][FUNC:FuncErrorReason][FILE:error_message_manage.cc][LINE:53]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         open device 0 failed, runtime result = 507033.[FUNC:ReportCallError][FILE:log_inner.cpp][LINE:161]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         ctx is NULL![FUNC:GetDevErrMsg][FILE:api_impl.cc][LINE:5372]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.py:572]         The argument is invalid.Reason: rtGetDevMsg execute failed, reason=[context pointer null]
(RayWorkerWrapper pid=118835) ERROR 02-26 09:45:37 worker_base.phttps://docs.ray.io/en/master/ray-observability/user-guides/configure-logging.html#log-deduplication
lugins: [repeated 15x across cluster] (Ray deduplicates logs by default. Set RAY_DEDUP_LOGS=0 to disable log deduplication, or see https://docs.ray.io/en/master/ray-observability/user-guides/configure-logging.html#log-deduplication for more options.)
(pid=118843) INFO 02-26 09:45:31 __init__.py:30] name=ascend, value=vllm_ascend:register [repeated 15x across cluster]
(pid=118843) INFO 02-26 09:45:31 __init__.py:32] all available plugins for group vllm.platform_plugins will be loaded. [repeated 15x across cluster]
(pid=118843) INFO 02-26 09:45:31 __init__.py:34] set environment variable VLLM_PLUGINS to control which plugins to load. [repeated 15x across cluster]
(pid=118843) INFO 02-26 09:45:31 __init__.py:42] plugin ascend loaded. [repeated 15x across cluster]
(pid=118843) INFO 02-26 09:45:31 __init__.py:187] No platform detected, vLLM is running on UnspecifiedPlatform [repeated 7x across cluster]
(pid=118843) WARNING 02-26 09:45:31 _custom_ops.py:19] Failed to import from vllm._C with ModuleNotFoundError("No module named 'vllm._C'") [repeated 7x across cluster]
(pid=118843) INFO 02-26 09:45:31 __init__.py:174] Platform plugin ascend is activated [repeated 7x across cluster]
(RayWorkerWrapper pid=118843) WARNING 02-26 09:45:32 utils.py:547] Overwriting environment variable ASCEND_RT_VISIBLE_DEVICES from '7' to '0,1,2,3,4,5,6,7' [repeated 6x across cluster]
ERROR 02-26 10:15:42 worker_base.py:572] Error executing method 'init_device'. This might cause deadlock in distributed execution.
ERROR 02-26 10:15:42 worker_base.py:572] Traceback (most recent call last):
ERROR 02-26 10:15:42 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm/worker/worker_base.py", line 564, in execute_method
ERROR 02-26 10:15:42 worker_base.py:572]     return run_method(target, method, args, kwargs)
ERROR 02-26 10:15:42 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm/utils.py", line 2208, in run_method
ERROR 02-26 10:15:42 worker_base.py:572]     return func(*args, **kwargs)
ERROR 02-26 10:15:42 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm_ascend/worker.py", line 181, in init_device
ERROR 02-26 10:15:42 worker_base.py:572]     init_worker_distributed_environment(self.parallel_config, self.rank,
ERROR 02-26 10:15:42 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm_ascend/worker.py", line 463, in init_worker_distributed_environment
ERROR 02-26 10:15:42 worker_base.py:572]     init_distributed_environment(parallel_config.world_size, rank,
ERROR 02-26 10:15:42 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm/distributed/parallel_state.py", line 960, in init_distributed_environment
ERROR 02-26 10:15:42 worker_base.py:572]     torch.distributed.init_process_group(
ERROR 02-26 10:15:42 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/torch/distributed/c10d_logger.py", line 83, in wrapper
ERROR 02-26 10:15:42 worker_base.py:572]     return func(*args, **kwargs)
ERROR 02-26 10:15:42 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/torch/distributed/c10d_logger.py", line 97, in wrapper
ERROR 02-26 10:15:42 worker_base.py:572]     func_return = func(*args, **kwargs)
ERROR 02-26 10:15:42 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/torch/distributed/distributed_c10d.py", line 1520, in init_process_group
ERROR 02-26 10:15:42 worker_base.py:572]     store, rank, world_size = next(rendezvous_iterator)
ERROR 02-26 10:15:42 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/torch/distributed/rendezvous.py", line 221, in _tcp_rendezvous_handler
ERROR 02-26 10:15:42 worker_base.py:572]     store = _create_c10d_store(
ERROR 02-26 10:15:42 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/torch/distributed/rendezvous.py", line 189, in _create_c10d_store
ERROR 02-26 10:15:42 worker_base.py:572]     return TCPStore(
ERROR 02-26 10:15:42 worker_base.py:572] torch.distributed.DistStoreError: Timed out after 1801 seconds waiting for clients. 1/8 clients joined.
Traceback (most recent call last):
  File "/work/.env/py310/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/work/.env/py310/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/entrypoints/openai/api_server.py", line 909, in <module>
    uvloop.run(run_server(args))
  File "/work/.env/py310/lib/python3.10/site-packages/uvloop/__init__.py", line 82, in run
    return loop.run_until_complete(wrapper())
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/work/.env/py310/lib/python3.10/site-packages/uvloop/__init__.py", line 61, in wrapper
    return await main
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/entrypoints/openai/api_server.py", line 873, in run_server
    async with build_async_engine_client(args) as engine_client:
  File "/work/.env/py310/lib/python3.10/contextlib.py", line 199, in __aenter__
    return await anext(self.gen)
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/entrypoints/openai/api_server.py", line 134, in build_async_engine_client
    async with build_async_engine_client_from_engine_args(
  File "/work/.env/py310/lib/python3.10/contextlib.py", line 199, in __aenter__
    return await anext(self.gen)
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/entrypoints/openai/api_server.py", line 158, in build_async_engine_client_from_engine_args
    engine_client = AsyncLLMEngine.from_engine_args(
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/engine/async_llm_engine.py", line 642, in from_engine_args
    engine = cls(
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/engine/async_llm_engine.py", line 592, in __init__
    self.engine = self._engine_class(*args, **kwargs)
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/engine/async_llm_engine.py", line 265, in __init__
    super().__init__(*args, **kwargs)
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/engine/llm_engine.py", line 271, in __init__
    self.model_executor = executor_class(vllm_config=vllm_config, )
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/executor/executor_base.py", line 260, in __init__
    super().__init__(*args, **kwargs)
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/executor/executor_base.py", line 49, in __init__
    self._init_executor()
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/executor/ray_distributed_executor.py", line 88, in _init_executor
    self._init_workers_ray(placement_group)
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/executor/ray_distributed_executor.py", line 343, in _init_workers_ray
    self._run_workers("init_device")
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/executor/ray_distributed_executor.py", line 464, in _run_workers
    self.driver_worker.execute_method(sent_method, *args, **kwargs)
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/worker/worker_base.py", line 573, in execute_method
    raise e
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/worker/worker_base.py", line 564, in execute_method
    return run_method(target, method, args, kwargs)
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/utils.py", line 2208, in run_method
    return func(*args, **kwargs)
  File "/work/.env/py310/lib/python3.10/site-packages/vllm_ascend/worker.py", line 181, in init_device
    init_worker_distributed_environment(self.parallel_config, self.rank,
  File "/work/.env/py310/lib/python3.10/site-packages/vllm_ascend/worker.py", line 463, in init_worker_distributed_environment
    init_distributed_environment(parallel_config.world_size, rank,
  File "/work/.env/py310/lib/python3.10/site-packages/vllm/distributed/parallel_state.py", line 960, in init_distributed_environment
    torch.distributed.init_process_group(
  File "/work/.env/py310/lib/python3.10/site-packages/torch/distributed/c10d_logger.py", line 83, in wrapper
    return func(*args, **kwargs)
  File "/work/.env/py310/lib/python3.10/site-packages/torch/distributed/c10d_logger.py", line 97, in wrapper
    func_return = func(*args, **kwargs)
  File "/work/.env/py310/lib/python3.10/site-packages/torch/distributed/distributed_c10d.py", line 1520, in init_process_group
    store, rank, world_size = next(rendezvous_iterator)
  File "/work/.env/py310/lib/python3.10/site-packages/torch/distributed/rendezvous.py", line 221, in _tcp_rendezvous_handler
    store = _create_c10d_store(
  File "/work/.env/py310/lib/python3.10/site-packages/torch/distributed/rendezvous.py", line 189, in _create_c10d_store
    return TCPStore(
torch.distributed.DistStoreError: Timed out after 1801 seconds waiting for clients. 1/8 clients joined.
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572] Error executing method 'init_device'. This might cause deadlock in distributed execution. [repeated 6x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572] Traceback (most recent call last): [repeated 6x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm/worker/worker_base.py", line 564, in execute_method [repeated 6x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572]     return run_method(target, method, args, kwargs) [repeated 6x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm/utils.py", line 2208, in run_method [repeated 6x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572]     return func(*args, **kwargs) [repeated 6x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/vllm_ascend/worker.py", line 173, in init_device [repeated 6x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572]     current_platform.set_device(self.device) [repeated 6x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572]   File "/work/.env/py310/lib/python3.10/site-packages/torch_npu/npu/utils.py", line 58, in set_device [repeated 12x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572]     torch.npu.set_device(device) [repeated 6x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572]     torch_npu._C._npu_setDevice(device_id) [repeated 6x across cluster]
(RayWorkerWrapper pid=118846) ERROR 02-26 09:45:38 worker_base.py:572] RuntimeError: Initialize:build/CMakeFiles/torch_npu.dir/compiler_depend.ts:226 NPU function error: c10_npu::SetDevice(device_id_), error code is 107001 [repeated 6x across cluster]

--- a/docs/source/tutorials.md
+++ b/docs/source/tutorials.md
@@ -245,6 +245,7 @@ export HCCL_IF_IP={local_ip}
 export GLOO_SOCKET_IFNAME={nic_name}
 export TP_SOCKET_IFNAME={nic_name}
 export ASCEND_RT_VISIBLE_DEVICES=0,1,2,3,4,5,6,7
+export RAY_EXPERIMENTAL_NOSET_ASCEND_RT_VISIBLE_DEVICES=1
 ray start --head --num-gpus=8
 
 # Worker node

